pwiz_aux/msrc/utility/vendor_api/Bruker/TimsData.cpp

```diff
@@ -862,21 +854,16 @@ void TimsSpectrum::getIsolationData(std::vector<IsolationInfo>& isolationInfo) c
     if (HasPasefPrecursorInfo())
     {
         const auto& info = GetPasefPrecursorInfo();
-        isolationInfo.resize(1, IsolationInfo{
-	                             info.isolationMz, IsolationMode_On, info.collisionEnergy,
-	                             info.intensity

```

Comment:
You reverted your intensity population here.

Reply:
Thank you for finding this. I will fix it.

Reply:
Done.